src/content/platform-integration/ios/setup.md

```diff
@@ -13,67 +12,102 @@ to run, build, and deploy Flutter apps for iOS devices.
 If you haven't set up Flutter already,
 visit and follow the [Get started with Flutter][] guide first.
 
-If you have already installed Flutter,
+If you've already installed Flutter,
 ensure that it's [up to date][].
 :::
 
 [Get started with Flutter]: /get-started/install
 [up to date]: /install/upgrade
 
-## Install Xcode
+## Set up iOS tooling {: #set-up-tooling}
 
 With Xcode, you can run Flutter apps on
 an iOS physical device or on the iOS Simulator.
 
-If you haven't done so already,
-install and set up [Xcode][].
+ 1. <h3>Install Xcode</h3>
 
-[Xcode]: https://developer.apple.com/xcode/
+    If you haven't done so already,
+    [install and set up the latest version of Xcode][xcode].
 
- 1. Configure Xcode's command-line tools:
+    If you've already installed Xcode,
+    update it to the latest version using the
+    same installation method you used originally.
+
+ 1. <h3>Set up Xcode command-line tools</h3>
+
+    To configure the Xcode command-line tools to use
+    the version of Xcode you installed,
+    run the following command in your preferred terminal:
 
     ```console
     $ sudo sh -c 'xcode-select -s /Applications/Xcode.app/Contents/Developer && xcodebuild -runFirstLaunch'
     ```
 
- 1. Sign the Xcode license agreement:
+    If you downloaded Xcode elsewhere or need to use a different version,
+    replace `/Applications/Xcode.app` with the path to there instead.
 
-    ```console
-    $ sudo xcodebuild -license
-    ```
+ 1. <h3>Agree to the Xcode licenses</h3>
+
+    After you've set up Xcode and configured its command-line tools,
+    agree to the Xcode licenses.
+
+    1. Open your preferred terminal.
+
+    1. Run the following command to review and sign the Xcode licenses.
 
- 1. Download iOS platform support and simulator runtimes:
+       ```console
+       $ sudo xcodebuild -license
+       ```
+
+    1. Read and agree to all necessary licenses.
+
+       Before agreeing to the terms of each license,
+       read each with care.
+
+       Once you've accepted all the necessary licenses successfully,
+       the command should output how to review the licenses.
```

Comment:
Is this sentence necessary? I'd consider cutting, I don't think it adds too much value :)